### ISSUE TITLE
[Truffle] Process:Module waitall needed for MRI tests

### ISSUE THREAD

**Issue by bjfish** [jruby/jruby#2827]
Error
```
NoMethodError: undefined method `waitall' for Process:Module
    test/mri/runner.rb:25:in `method_missing': undefined method `waitall' for Process:Module (NoMethodError)
    	from test/mri/runner.rb:25:in `after_teardown'

```

**Comment by bjfish** [comment]
The change in runner.rb can be used to workaround this issue to run `jt test mri` tests:
https://gist.github.com/bjfish/1be84623da4a746f4d16

**Comment by chrisseaton** [comment]
Fixed in 6fcb945dff9e465aba861e6cb58e744c27610b27
